### PR TITLE
[FEATURE] Migrate calendars controller test.rb

### PR DESCRIPTION
**Pull request by hschink**
Implements [ticket 2057](https://www.openproject.org/issues/2057)

### CODE REVIEW

**Comment by ulferts** [comment]
Travis doesn't like it:

> Failing Scenarios:
> 
> cucumber features/projects/show.feature:29 # Scenario: Calendar link in the 'tickets box' should work when calendar is activated

File: config/routes.rb [hunk 1]
```diff
@@ -206,10 +206,6 @@
     # work as a catchall for everything under /wiki
     get 'wiki' => "wiki#show"
 
-    namespace :issues do
-      resources :calendar, :controller => 'calendars', :only => [:index]
```
Comment (ulferts) [inline config/routes.rb:210]: You deleted the route for 

> /projects/:project_id/issues/calendar

 and added one for

> /work_packages/calendar

There is still a route defined for

> /issues/calendar

while the one for 

> /projects/:project_id/work_packages/calendar

is missing. The test for that route is missing, too.

This results in the url for the calendar menu entry beeing displayed as e.g.

> /work_packages/calendar?project_id=openproject

**Comment by ulferts** [comment]
There is an occurrence of "issues/calendar" in app/views/welcome/robots.text.erb

While this file appears to be maintained less good, this is a chance to get better.

**Comment by ulferts** [comment]
Works as expected, otherwise.

Fix the points mentioned and give me a ping. I will merge it.

**Comment by hschink** [comment]
@ulferts Thanks for the hint! You seem to see everything! :mag: And yeah, :bell: PING!

**Pull request merged**